### ISSUE TITLE
Predicates generated incorrectly when using Graphs

### ISSUE THREAD

**Issue by amohoste** [RMLio/rmlmapper-java#2]
Using the following yarrrml mapping file:

```
prefixes:
  # Predefined
  gbfs: "http://example.com/terms#"
  geo: "http://www.w3.org/2003/01/geo/wgs84_pos#"
  ex: "http://example.com/"
  
sources:
  station_status:
    access: https://api-core.bikeshare.ae/gbfs/ar/station_status.json
    referenceFormulation: jsonpath
    iterator: $.data.stations[*]

mappings:
  station_status:
    graphs: ex:mygraph
    sources: station_status
    s: http://example.org/station/$(station_id)
    po:
      - [gbfs:num_docks_disabled, $(num_docks_disabled), xsd:integer]
      - [gbfs:num_docks_available, $(num_docks_available), xsd:integer]
```

I generated the RML file shown below through matey.
```
@prefix rr: <http://www.w3.org/ns/r2rml#>.
@prefix rml: <http://semweb.mmlab.be/ns/rml#>.
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
@prefix ql: <http://semweb.mmlab.be/ns/ql#>.
@prefix map: <http://mapping.example.com/>.

map:gm_0 a rr:GraphMap;
    rr:template "http://example.com/mygraph".
map:map_station_status_0 rml:logicalSource map:source_0;
    a rml:TriplesMap;
    rdfs:label "station_status";
    rr:subjectMap map:s_0;
    rr:predicateObjectMap map:pom_0, map:pom_1.
map:om_0 a rr:ObjectMap;
    rr:template "{num_docks_disabled}";
    rr:termType rr:Literal;
    rr:datatype <http://www.w3.org/2001/XMLSchema#integer>.
map:om_1 a rr:ObjectMap;
    rr:template "{num_docks_available}";
    rr:termType rr:Literal;
    rr:datatype <http://www.w3.org/2001/XMLSchema#integer>.
map:pm_0 a rr:PredicateMap;
    rr:constant <http://example.com/terms#num_docks_disabled>.
map:pm_1 a rr:PredicateMap;
    rr:constant <http://example.com/terms#num_docks_available>.
map:pom_0 a rr:PredicateObjectMap;
    rr:predicateMap map:pm_0;
    rr:objectMap map:om_0.
map:pom_1 a rr:PredicateObjectMap;
    rr:predicateMap map:pm_1;
    rr:objectMap map:om_1.
map:s_0 a rr:SubjectMap;
    rr:template "http://example.org/station/{station_id}";
    rr:graphMap map:gm_0.
map:source_0 a rml:LogicalSource;
    rdfs:label "station_status";
    rml:source "https://api-core.bikeshare.ae/gbfs/ar/station_status.json";
    rml:iterator "$.data.stations[*]";
    rml:referenceFormulation ql:JSONPath.
```

When using matey, the linked data is correctly generated as following:
```
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
@prefix sd: <http://www.w3.org/ns/sparql-service-description#>.
@prefix gbfs: <http://example.com/terms#>.
@prefix geo: <http://www.w3.org/2003/01/geo/wgs84_pos#>.
@prefix ex: <http://example.com/>.

ex:mygraph {
<http://example.org/station/3> gbfs:num_docks_disabled "0"^^xsd:integer;
    gbfs:num_docks_available "13"^^xsd:integer
}
ex:mygraph {
<http://example.org/station/4> gbfs:num_docks_disabled "0"^^xsd:integer;
    gbfs:num_docks_available "5"^^xsd:integer
}
ex:mygraph {
<http://example.org/station/5> gbfs:num_docks_disabled "0"^^xsd:integer;
    gbfs:num_docks_available "9"^^xsd:integer
}
ex:mygraph {
<http://example.org/station/6> gbfs:num_docks_disabled "0"^^xsd:integer;
    gbfs:num_docks_available "5"^^xsd:integer
}
...
```

However, when I try to use rmlmapper-java to generate linked data from the generated RML file using the following command: `java -jar rmlmapper-0.1.4-r19-jar-with-dependencies.jar -m rml.txt -o outfile`. The predicates are incorrect (same as subject, see below).

```
<http://example.org/station/3> <http://example.org/station/3> "0"^^<http://www.w3.org/2001/XMLSchema#integer> <http://example.com/mygraph>.
<http://example.org/station/3> <http://example.org/station/3> "13"^^<http://www.w3.org/2001/XMLSchema#integer> <http://example.com/mygraph>.
<http://example.org/station/4> <http://example.org/station/4> "0"^^<http://www.w3.org/2001/XMLSchema#integer> <http://example.com/mygraph>.
<http://example.org/station/4> <http://example.org/station/4> "5"^^<http://www.w3.org/2001/XMLSchema#integer> <http://example.com/mygraph>.
<http://example.org/station/5> <http://example.org/station/5> "0"^^<http://www.w3.org/2001/XMLSchema#integer> <http://example.com/mygraph>.
<http://example.org/station/5> <http://example.org/station/5> "9"^^<http://www.w3.org/2001/XMLSchema#integer> <http://example.com/mygraph>.
<http://example.org/station/6> <http://example.org/station/6> "0"^^<http://www.w3.org/2001/XMLSchema#integer> <http://example.com/mygraph>.
<http://example.org/station/6> <http://example.org/station/6> "5"^^<http://www.w3.org/2001/XMLSchema#integer> <http://example.com/mygraph>.
<http://example.org/station/7> <http://example.org/station/7> "0"^^<http://www.w3.org/2001/XMLSchema#integer> <http://example.com/mygraph>.
<http://example.org/station/7> <http://example.org/station/7> "8"^^<http://www.w3.org/2001/XMLSchema#integer> <http://example.com/mygraph>.
<http://example.org/station/8> <http://example.org/station/8> "0"^^<http://www.w3.org/2001/XMLSchema#integer> <http://example.com/mygraph>.
<http://example.org/station/8> <http://example.org/station/8> "8"^^<http://www.w3.org/2001/XMLSchema#integer> <http://example.com/mygraph>.
<http://example.org/station/9> <http://example.org/station/9> "0"^^<http://www.w3.org/2001/XMLSchema#integer> <http://example.com/mygraph>.
<http://example.org/station/9> <http://example.org/station/9> "16"^^<http://www.w3.org/2001/XMLSchema#integer> <http://example.com/mygraph>.
...
```

When not using graphs, I get the correct output:
```
<http://example.org/station/3> <http://example.com/terms#num_docks_disabled> "0"^^<http://www.w3.org/2001/XMLSchema#integer>.
<http://example.org/station/3> <http://example.com/terms#num_docks_available> "13"^^<http://www.w3.org/2001/XMLSchema#integer>.
<http://example.org/station/4> <http://example.com/terms#num_docks_disabled> "0"^^<http://www.w3.org/2001/XMLSchema#integer>.
<http://example.org/station/4> <http://example.com/terms#num_docks_available> "5"^^<http://www.w3.org/2001/XMLSchema#integer>.
<http://example.org/station/5> <http://example.com/terms#num_docks_disabled> "0"^^<http://www.w3.org/2001/XMLSchema#integer>.
<http://example.org/station/5> <http://example.com/terms#num_docks_available> "9"^^<http://www.w3.org/2001/XMLSchema#integer>.
<http://example.org/station/6> <http://example.com/terms#num_docks_disabled> "0"^^<http://www.w3.org/2001/XMLSchema#integer>.
<http://example.org/station/6> <http://example.com/terms#num_docks_available> "5"^^<http://www.w3.org/2001/XMLSchema#integer>.
<http://example.org/station/7> <http://example.com/terms#num_docks_disabled> "0"^^<http://www.w3.org/2001/XMLSchema#integer>.
<http://example.org/station/7> <http://example.com/terms#num_docks_available> "8"^^<http://www.w3.org/2001/XMLSchema#integer>.
<http://example.org/station/8> <http://example.com/terms#num_docks_disabled> "0"^^<http://www.w3.org/2001/XMLSchema#integer>.
<http://example.org/station/8> <http://example.com/terms#num_docks_available> "8"^^<http://www.w3.org/2001/XMLSchema#integer>.
<http://example.org/station/9> <http://example.com/terms#num_docks_disabled> "0"^^<http://www.w3.org/2001/XMLSchema#integer>.
<http://example.org/station/9> <http://example.com/terms#num_docks_available> "16"^^<http://www.w3.org/2001/XMLSchema#integer>.
<http://example.org/station/10> <http://example.com/terms#num_docks_disabled> "0"^^<http://www.w3.org/2001/XMLSchema#integer>.
<http://example.org/station/10> <http://example.com/terms#num_docks_available> "6"^^<http://www.w3.org/2001/XMLSchema#integer>.
<http://example.org/station/11> <http://example.com/terms#num_docks_disabled> "0"^^<http://www.w3.org/2001/XMLSchema#integer>.
<http://example.org/station/11> <http://example.com/terms#num_docks_available> "4"^^<http://www.w3.org/2001/XMLSchema#integer>.
<http://example.org/station/12> <http://example.com/terms#num_docks_disabled> "0"^^<http://www.w3.org/2001/XMLSchema#integer>.
```

**Comment by pheyvaer** [comment]
Hi @amohoste,

This should be fixed in [version 0.1.5](https://github.com/RMLio/rmlmapper-java/releases/tag/v0.1.5). Could you have a look and see if this issue is also fixed on your side?

**Comment by amohoste** [comment]
Issue is fixed, thanks!
